addons/sale_project/models/sale_order_line.py

```diff
@@ -150,6 +150,7 @@ def _timesheet_create_project_prepare_values(self):
             'sale_line_id': self.id,
             'active': True,
             'company_id': self.company_id.id,
+            'allow_billable': True,
```

Comment:
Could you rephrase the commit like this?
```
[FIX] sale_project: create project from SOL with allow_billable
This commit purpose is to ensure that projects created from a SOL
are created with `allow_billable` set to `True`

task-2992614
```
Thanks and sorry for being such a pain 🤣